src/ToggleButton.js

```diff
@@ -97,19 +90,17 @@ class ToggleButton extends React.Component {
           autoComplete="off"
           checked={!!checked}
           disabled={!!disabled}
-          onFocus={this.handleFocus}
-          onBlur={this.handleBlur}
+          onFocus={toggleFocus}
```

Comment:
use separate `handleFocus` and `handleBlur`. those callbacks need fewer dependencies, plus it's more explicit – the action on focus is specifically to set focused to true, not to toggle the state